sdf.md

```diff
@@ -165,34 +163,33 @@ Properties are used to model elements of state.
 
 - Qualities of odmProperty
 
-| Quality | Type | Optional | Description |
-|---|---|---|---|
-|id| integer | yes | internal unique identifier for the definition |
-|name|string|yes|human readable name|
-|description|string|yes|human readable description|
-|title|string|yes|human readable title to display|
-|optional| boolean|yes|defines whether this element is optional in an implementation|
+| Quality | Type | Required | Description | Default |
+|---|---|---|---|---|
+|id| integer | yes | internal unique identifier for the definition | N/A |
+|name|string|yes|human readable name| N/A |
```

Comment:
Now name, desc, title are required for property but not for object. Maybe we want to keep these consistent for now and then discuss in a separate issue what should be required and what not to have all that documented in a single place.